src/index.js

```diff
@@ -161,6 +161,54 @@ class EventPortal {
     }
   }
 
+  async getApplications(params=null) {
+    try {
+        console.log(`Fetching Applications`)
+        let url = `applications`
+        url = url + (params ? `?${params}` : '');
+        const response = await this.api(this.token, 'GET', url, null)
+        return response;
+    } catch (error) {
+      throw new Error(error)
+    }
+  }
+
+  async getApplicationByID(applicationId, params=null) {
+    try {
+        console.log(`Fetching Application By ID`)
+        let url = `applications/${applicationId}`
+        url = url + (params ? `?${params}` : '');
+        const response = await this.api(this.token, 'GET', url, null)
+        return response;
+    } catch (error) {
+        throw new Error(error)
+    }
+  }  
+
+  async getApplicationVersions(applicationId, params=null) {
```

Comment:
shouldnt this return an array of applicationID versions? instead of returning the raw response?